pathfind/Map.cpp

```diff
@@ -816,7 +863,8 @@ bool Map::ZoneAndArea(const math::Vertex& position, unsigned int& zone,
         {position.X, position.Y, tile->second->m_bounds.getMinimum().Z}};
 
     unsigned int localZone, localArea;
-    auto const rayResult = RayCast(ray, tiles, &localZone, &localArea);
+    const auto doodads = false;
```

Comment:
This doesn't seem to be referenced anywhere else.  You can just put `false` into the `RayCast()` call.